datatype.go

```diff
@@ -32,27 +32,45 @@ const (
 	unSupportedType
 )
 
-var snowflakeTypes = [...]string{"FIXED", "REAL", "TEXT", "DATE", "VARIANT",
-	"TIMESTAMP_LTZ", "TIMESTAMP_NTZ", "TIMESTAMP_TZ", "OBJECT", "ARRAY",
-	"BINARY", "TIME", "BOOLEAN", "NULL", "SLICE", "CHANGE_TYPE", "NOT_SUPPORTED"}
+var stringToSnowflakeType = map[string]snowflakeType{
```

Comment:
I wonder if we can rename it to be more precise. Because the first one is actual snowflake type and the second is the driver (go?) type. WDYT?

Reply:
Changed names to be more descriptive